### DISCUSSION TITLE
Language packs - looking for a anyone to do translations

### DISCUSSION THREAD

**Discussion by amnuts** [amnuts/opcache-gui#89] [General]
I'm wanting to introduce the ability to rebuild a different language page into the gui via the build command.

Would anyone out there be willing to translate some English text to any other language?

If so, please give me a shout!

**Comment by bbalet** [comment]
I can help you with French language.
BTW it looks like you have strings into the backend too.

**Reply by amnuts** [reply]
Hey @bbalet, that's really awesome of you, thanks!

I have this [languages ](https://github.com/amnuts/opcache-gui/tree/languages) branch which is currently in progress.  The plan is that when I have at least one language file, I'll update the `build/build.php` file to allow it to be selected and rebuild the `index.php` with that language (so that I can still retain the, "you only need one file to run it" methodology).

In that branch, you'll find the file `build/_languages/example.json` which basically has all the strings.  The key is the English version which matches what's in the UI and the value is what needs replacing - which in the example file is just blank.  If a value is empty or the index doesn't exist for a translation then it'll just use the English version.

So if you could fork the branch, copy the `example.json` file and call it `fr.json`, make your changes and create a PR against `langauges` branch, that'd be really awesome!

Many thanks!

**Comment by bbalet** [comment]
OK I'm on it

**Reply by amnuts** [reply]
Just incase you hadn't seen it, @bbalet, as the PR got closed when merged: https://github.com/amnuts/opcache-gui/pull/91#issuecomment-1188274179
